src/Pages/ContentListTable/components/AddContent/AddContent.tsx

```diff
@@ -1,4 +1,5 @@
 import {
+    AlertVariant,
```

Comment:
there's also a linter warnining for this being unused

Reply:
strange, the linter was passing, it caught others of these

Reply:
but good catch!